chain/stmgr/forks.go

```diff
@@ -603,7 +724,7 @@ func resetGenesisMsigs(ctx context.Context, sm *StateManager, store adt0.Store,
 				return xerrors.Errorf("reading multisig state: %w", err)
 			}
 
-			currState.StartEpoch = build.UpgradeLiftoffHeight
+			currState.StartEpoch = epoch
```

Comment:
Ok, I've fixed this, but we need a better fix.

Basically, I'm trying to avoid referencing these constants because I allow changing the upgrade schedule via state-manager parameter.

Currently, I'm just requiring that the liftoff epoch be beyond the ignition upgrade epoch.